content/blog/a-simple-guide-to-noatarizing-your-java-application/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: A simple guide to notarzing your Java application
+date: "2020-05-21T12:00:00+00:00"
+author: georgeadams
+description: Apple has recently changed the requirements for applications to install on macOS 10.15 and above.
+featuredImage: "./catalina.jpeg"
+tags:
+  - macOS
+  - Catalina
+  - Notarization
+---
+
+Apple has recently changed the requirements for applications to install on macOS 10.15 and above.
+
+![MacOS Catalina background](./catalina.jpeg)
+
+## Summary
+Apple has recently changed the requirements for applications to install on macOS 10.15 and above. The change requires developers to notarize the application before it gets shipped. Notarization involves submitting the application to Apple to be scanned and generates a JSON report with any issues. More information about notarization can be found [here](https://developer.apple.com/documentation/security/notarizing_your_app_before_distribution).
```

Comment:
Instead of "here":

```
More information about notarization can be found in the article [Notarizing macOS Software Before Distribution](https://developer.apple.com/documentation/security/notarizing_your_app_before_distribution) on the Apple website.
```